spec/xcpretty/formatters/simple_spec.rb

```diff
@@ -20,6 +20,11 @@ module XCPretty
         "> Building Pods/The Spacer [Debug]"
       end
 
+      it "formats analyze target/project/configuration with target" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.